Note port for rpm packages in post install section

scollazo opened the pull request:
Closes artefactual/archivematica-docs#92

scollazo committed:
Note port for rpm packages in post install section

Closes artefactual/archivematica-docs#92

sevein reviewed (changes requested):
Nitpicks:
- Changes don't use 80cols.
- Things like 8001, 81, etc... could be inline literals for better readability.
- "RPM packages use port 8001" is a sentence, "(RPM packages use port 81)" however use parenthesis.

scollazo commented:
Changed to 80 columns, and added brackets  to both sentences.

Reading the section, I prefer to don't use inline, as there isn't any inline port in other paragraphs of the section.

sevein commented on admin-manual/installation/installation.rst:
```diff
@@ -758,8 +758,8 @@ above, follow these steps to complete the configuration of your new server.
-   *test* with the password *test*: http://localhost:8000 (or use the IP
-   address of the machine you have been installing on).
+   *test* with the password *test*: http://localhost:8000, or use the IP address
+   of the machine you have been installing on ( RPM packages use port 8001 ).
```
Nitpick: typographical error, should read `(RPM packages use port 8001)` instead of `( RPM packages use port 8001 )`.

sevein commented on admin-manual/installation/installation.rst:
```diff
@@ -784,9 +784,10 @@ above, follow these steps to complete the configuration of your new server.
-   web browser: http://localhost (again, use the IP address of the machine you
-   have been installing on). When prompted, enter the URL of the Storage
-   Service, the name of the administrative user, and that user's API key.
+   web browser: http://localhost. Again, you can use the IP address of the machine you
```
^ this line has 86 cols

scollazo committed:
Wrap to 80 columns and remove extra spaces

sevein reviewed (approved):
Thanks.
